griptape/artifacts/action_artifact.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING, Optional
+
+from attrs import define, field
+
+from griptape.artifacts import BaseArtifact
+from griptape.mixins import SerializableMixin
+
+if TYPE_CHECKING:
+    from griptape.tools import BaseTool
+
+
+@define()
+class ActionArtifact(BaseArtifact, SerializableMixin):
```

Comment:
After adding the "MessageContent" classes, do we still need an `ActionArtifact` class? Can you put an `Action` directly onto the MessageContent classes now?

(Raising as per offline discussion)

Reply:
Probably not. But I think an argument can be made that since an LLM can generate Actions, it makes sense as an Artifact. Also, splitting the Message Content types into "Artifact Message Content" "Non Artifact Message Content"  would be a very tedious refactor at this stage in the PR 😅